### PR TITLE
Bootstrap

### PR DESCRIPTION
**Pull request by maxmodlol**

### CODE REVIEW

File: form.html [hunk 1]
```diff
@@ -0,0 +1,79 @@
+<!DOCTYPE html>
```
Comment (Abdallah-Via) [inline form.html:1]: Here you did the same mistake, this file should be in different branch, not here

File: task-json/q2.js [hunk 2]
```diff
@@ -1,27 +1,22 @@
```
Comment (Abdallah-Via) [inline task-json/q2.js:1]: Here you did the same mistake, this file should be in different branch, not here

File: bootstrap.html [hunk 3]
```diff
@@ -0,0 +1,107 @@
...
+        </label>
+      </div>
+      <div class="form-check">
+        <input class="form-check-input" type="radio" name="exampleRadios" id="exampleRadios2" value="option2">
+        <label class="form-check-label" for="exampleRadios2">
+          Female
+        </label>
+      </div>
+      <div class="form-check">
+        <p > Please Select Your Couurses</p>
```
Comment (Abdallah-Via) [inline bootstrap.html:62]: This will breaks the ui flow, please check it in ui & fix it

File: bootstrap.html [hunk 4]
```diff
@@ -0,0 +1,107 @@
...
+
+        <input class="form-check-input" type="checkbox" value="" id="defaultCheck1">
```
Comment (Abdallah-Via) [inline bootstrap.html:64]: you have to add a value here, to use it when submitting the form

File: bootstrap.html [hunk 5]
```diff
@@ -0,0 +1,107 @@
...
+        <label class="form-check-label" for="defaultCheck1">
+          Java
+        </label>
+      </div>
+      <div class="form-check">
+
+        <input class="form-check-input" type="checkbox" value="" id="defaultCheck2" >
```
Comment (Abdallah-Via) [inline bootstrap.html:71]: same as above for value

File: bootstrap.html [hunk 6]
```diff
@@ -0,0 +1,107 @@
...
+        <label class="form-check-label" for="defaultCheck2">
+          C++
+        </label>
+      </div>
+      <div class="form-check">
+
+        <input class="form-check-input" type="checkbox" value="" id="defaultCheck3" >
```
Comment (Abdallah-Via) [inline bootstrap.html]: same as above for value

File: bootstrap.html [hunk 7]
```diff
@@ -0,0 +1,107 @@
...
+          <input type="color" class="form-control form-control-color" id="exampleColorInput" value="#008000" title="Choose your color">
+        </section>
+      </div>
+      <div class="form-group">
+        <label for="formFile" class="form-label">Image </label>
+        <input class="form-control" type="file" id="formFile" accept=".jpg,.gif,.png">
+      </div>
+      <div class="form-check">
+      <p > Please Select Your Gender</p>
+        <input class="form-check-input" type="radio" name="exampleRadios" id="exampleRadios1" value="option1" checked>
```
Comment (Abdallah-Via) [inline bootstrap.html]: you have to use a readable name

File: bootstrap.html [hunk 8]
```diff
@@ -0,0 +1,107 @@
...
+        <label class="form-check-label" for="exampleRadios1">
+          Male
+        </label>
+      </div>
+      <div class="form-check">
+        <input class="form-check-input" type="radio" name="exampleRadios" id="exampleRadios2" value="option2">
```
Comment (Abdallah-Via) [inline bootstrap.html]: you have to use a readable name in value

File: bootstrap.html [hunk 9]
```diff
@@ -0,0 +1,107 @@
...
+
+        <input placeholder="Selected date" type="text" id="date-picker-example" class="form-control datepicker">
+      </div>
+      <div class="form-group">   
+        <label for="replyNumber">Salary</label>
+                   
+        <input type="number" class="form-control" id="replyNumber" min="0" data-bind="value:replyNumber" />
+      </div>
+      <div class="form-group">
+        <section class="form-group">
```
Comment (Abdallah-Via) [inline bootstrap.html:51]: This will create a full width icon, please fix the ui part

File: bootstrap.html [hunk 10]
```diff
@@ -9,36 +9,48 @@
...
+    }
+    th, td{
+        padding: 10px 20px;
+        border: 1px solid #000;
+    }
+</style>
+  
 </head>
-<body>
+<body id="body">
```
Comment (Abdallah-Via) [inline bootstrap.html:25]: You don't need to create id for body, there is no other body tag in normal cases

File: bootstrap.html [hunk 11]
```diff
@@ -104,4 +116,98 @@
...
+ }
+
+ let table = document.createElement('table');
+let thead = document.createElement('thead');
+let tbody = document.createElement('tbody');
+
+table.appendChild(thead);
+table.appendChild(tbody);
+
+document.getElementById('body').appendChild(table);
```
Comment (Abdallah-Via) [inline bootstrap.html:152]: you can use getElementsByTagName

File: bootstrap.html [hunk 12]
```diff
@@ -104,4 +116,98 @@
...
\ No newline at end of file
+</script>
+<script type="text/javascript">
+  $('#save').on('click', function() {
+  var first_name=$('#firstName').val();
+  var last_name=$('#lastName').val();
+  var date=$('#date-picker').val();
+ var salary=$('#salary').val();
+ var color= $('#color').val();
+ var textarea=$('#Textarea').val();
```
Comment (Abdallah-Via) [inline bootstrap.html]: try to create an array of { id, label }, then loop the array

File: bootstrap.html (hunk 10 above)
Comment (maxmodlol) [reply bootstrap.html:25]: i give it just to call it by id in javascript so i can append the created html element

Comment (maxmodlol) [reply bootstrap.html:25]: i give it just to call it by id in javascript so i can append the created html element

File: bootstrap.html [hunk 13]
```diff
@@ -47,47 +59,47 @@
...
+        <label class="form-check-label" for="female_option">
           Female
         </label>
       </div>
       <div class="form-check">
         <p > Please Select Your Couurses</p>
 
-        <input class="form-check-input" type="checkbox" value="" id="defaultCheck1">
-        <label class="form-check-label" for="defaultCheck1">
+        <input class="form-check-input" type="checkbox" name="check_box" value="JAVA" id="check_v1">
```
Comment (Abdallah-Via) [inline bootstrap.html:76]: instead of naming check_v1,v2,v3 use readable names

File: bootstrap.html (hunk 10 above)
Comment (Abdallah-Via) [reply bootstrap.html:25]: I understand it, but you can use getElementsByTagName

**Pull request closed without merging**